lightpath/tests/conftest.py

```diff
@@ -172,3 +175,24 @@ def simulated_lcls():
 def lcls_ctrl(lcls_client: happi.Client):
     print(f'first item: {lcls_client.search()[0]}')
     return LightController(lcls_client)
+
+
+@pytest.fixture(scope='function')
+def cfg():
+    cfg_path = os.path.join(os.path.dirname(__file__), 'conf.yml')
+    with open(cfg_path, 'r') as f:
+        cfg = yaml.safe_load(f)
+
+    return cfg
+
+
+@contextmanager
+def cli_args(args):
+    """
+    Context manager for running a block of code with a specific set of
+    command-line arguments.
+    """
+    prev_args = sys.argv
```

Comment:
Unnecessary to change as the end result is the same, but fyi - `parse_args` takes (I think) an `args=None` parameter, which can be a replacement for looking at `sys.argv`. Then your entrypoint would turn into `entrypoint(args=None)`. It's a bit cleaner than monkeypatching.